src/App/Fossa/Analyze.hs

```diff
@@ -282,6 +283,18 @@ analyze cfg = Diag.context "fossa-analyze" $ do
       shouldAnalyzePathDependencies = resolvePathDependencies $ Config.experimental cfg
       allowedTactics = Config.allowedTacticTypes cfg
 
+  manualSrcUnits <-
```

Comment:
Moving call to extract / validate fossa-deps earlier on in the analyze work flow